opm/grid/GraphOfGrid.cpp

```diff
@@ -271,6 +271,9 @@ int GraphOfGrid<Grid>::wellID (int gID) const
 template<typename Grid>
 void GraphOfGrid<Grid>::mergeWellIndices(const std::set<int>& well)
 {
+    if (!well.empty())
```

Comment:
i doubt you want to return if we are not empty.

Reply:
It should be when it is empty, I'll correct that